theme/templates/accounts/sign_up_form.html

```diff
@@ -47,13 +67,31 @@
         });
 
         $("#signup").click(function() {
-            var username = $("#username").val();
-            var firstName = $("#firstname").val();
-            var lastName = $("#lastname").val();
-            var email = $("#email").val();
-            var password = $("#password").val();
-            var pwconfirm = $("#pwconfirm").val();
-
+            var password = $("#id_password1").val();
+            var pwconfirm = $("#id_password2").val();
+            var fieldsToCheck = ["#id_username", "#id_email", "#id_password1", "#id_password2",
+                "#recaptcha_response_field"];
+            var fieldNames = [ 'User name', 'Email', 'Password', 'Password (again)', 'ReCaptcha'];
```

Comment:
@pkdash I suggest to use a dictionary to keep fieldsToCheck and fieldNames in pair rather than keeping them in two lists assuming the same order, just a suggestion for you to consider.